### ISSUE TITLE
onSnap firing when Interactable.View is not on snapPoint

### ISSUE THREAD

**Issue by matzewagner** [wix/react-native-interactable#101]
When I'm dragging an Interactable.View on iOS, onSnap is firing when I stop the dragging motion, but the View is not on a snapPoint. This makes onSnap behave pretty much the same way as onStop. Is this intended?

**Comment by rogchap** [comment]
Is there a workaround to this? I'm facing the same issue.

**Comment by rogchap** [comment]
This was my workaround:

```javascript
_onDrag = (e) => {
  const { state } = e.nativeEvent;
  this._isDragging = state === 'start'; // don't use state as you don't want to re-render
}

_onSnap = (e) => {
   const { index } = e.nativeEvent;
   if (index === 0 && !this._isDragging) {
     // Do stuff here
   }
}
```

**Comment by Inbal-Tish** [comment]
resolved in PR #254
